src/microbe_stage/Membrane.tscn

```diff
@@ -1,16 +1,16 @@
-[gd_scene load_steps=8 format=3 uid="uid://jgcbwcrqbblv"]
+[gd_scene load_steps=11 format=3 uid="uid://jgcbwcrqbblv"]
 
 [ext_resource type="Script" path="res://src/microbe_stage/Membrane.cs" id="1"]
 [ext_resource type="Shader" path="res://shaders/Membrane.gdshader" id="2"]
 [ext_resource type="Texture2D" uid="uid://c3fla17itmoba" path="res://assets/textures/FresnelGradient.png" id="3"]
 [ext_resource type="Texture2D" uid="uid://lei41d7q7tgk" path="res://assets/textures/FresnelGradientDamaged.png" id="4"]
 [ext_resource type="Texture2D" uid="uid://baxuoyeo83r2u" path="res://assets/textures/dissolve_noise.tres" id="5"]
+[ext_resource type="Shader" path="res://shaders/EngulfEffect.gdshader" id="6_w5050"]
 
 [sub_resource type="BoxMesh" id="1"]
 size = Vector3(2, 0.539, 2)
 
-[sub_resource type="ShaderMaterial" id="2"]
-resource_local_to_scene = true
```

Comment:
This needs to be kept true, this is assumed by the membrane code for it to safely modify the shader parameters. Also the new shader material you added also needs to be marked scene local.